native/src/routes/Licenses.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { FlatList, View } from 'react-native'
+import styled from 'styled-components/native'
+
+import { License, parseLicenses, JsonLicenses } from 'api-client/src/utils/licences'
+
+import Caption from '../components/Caption'
+import Layout from '../components/Layout'
+import Touchable from '../components/Touchable'
+import useSnackbar from '../hooks/useSnackbar'
+import openExternalUrl from '../utils/openExternalUrl'
+import { reportError } from '../utils/sentry'
+
+const PadView = styled.View`
```

Comment:
You could just apply the styling directly on the Touchable:
```suggestion
const StyledTouchable = styled.Touchable`
```

Reply:
how? I thought I have to use the PadView to get a correct alignment